website/docs/docs/build/metricflow-commands.md

```diff
@@ -429,9 +424,27 @@ mf query --metrics order_total --group-by metric_time,is_food_order --limit 10 -
 
 </TabItem>
 
+<TabItem value="eg6" label=" Saved queries">
 
-</Tabs>
+You can use this for frequently used queries. Replace `<name>` with the name of your [saved query](/docs/build/saved-queries). 
 
+**Query**
+```bash
+dbt sl query --saved-query <name> # In dbt Cloud
+
+mf query --saved-query <name> # In dbt Core
+```
+
+For example, if you use dbt Cloud and have a saved query named `new_customer_orders`, you would run `dbt sl query --saved-query new_customer_orders`.
+
+:::info A note on querying saved queries
+When querying [saved queries](/docs/build/saved-queries),you can use parameters such as `where`, `limit`, `order`, `compile`, and so on. However, note that `metric` or `group_by` parameters aren't available in this context.
+
+(SHOULD WE SAY WHY TEY CAN'T USE IT? BC IT'S PREDEFINED RIGHT? SHOULD WE LINK TO [THESE PARAMETERS](https://docs.getdbt.com/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-values)?)
```

Comment:
Yea its already defined in the saved query, however I think a simple mention is fine as even if they try it would just error with the relevant error message

Reply:
We should add saved query to that table of parameters too!

Reply:
Yeah, good call, Will!